js/cryptocom.js

```diff
@@ -139,6 +139,12 @@ module.exports = class cryptocom extends Exchange {
                             'private/subaccount/get-sub-accounts': 10 / 3,
                             'private/subaccount/get-transfer-history': 10 / 3,
                             'private/subaccount/transfer': 10 / 3,
+                            'private/otc/get-otc-user': 10 / 3,
+                            'private/otc/get-instruments': 10 / 3,
+                            'private/otc/request-quote': 100,
```

Comment:
@sc0Vu can you please elaborate a bit on how you get the value of 100 for this endpoint cost?

- https://exchange-docs.crypto.com/spot/index.html#rate-limits

Reply:
It supposed to be 1 request per second ((1000/1) / 10 = 100):

path | rate limit
-- | --
private/otc/request-quote | 1 requests per second
private/otc/accept-quote | 1 requests per second